cmd/cerbosctl/get/internal/policy/policy.go

```diff
@@ -57,24 +53,22 @@ func List(c client.AdminClient, cmd *cobra.Command, filters *flagset.Filters, fo
 				return fmt.Errorf("error while requesting policy: %w", err)
 			}
 
-			wp := make([]policy.Wrapper, len(policies))
+			keyPolicyPairs := make([]KeyPolicyPair, len(policies))
```

Comment:
It looks like we have to do some extra work on the client side to match-up the "store identifier" to the policy. That feels like a problem with the API to me. When possible, we should strive to make the API pleasant to use. So here's my suggestion:

- Let's add a new field named `store_identifer` to [`Policy.Metadata`](https://github.com/cerbos/cerbos/blob/ff338f84dad041d0676ed9abaf2b39e45eb54719/api/public/cerbos/policy/v1/policy.proto#L30-L33) to hold the store identifier.  
- In the `LoadPolicy` methods of the storage implementations, make sure to populate the `store_identifier` metadata field of the  policy. (See the `policy.WithMetadata` function we use in the [index builder](https://github.com/cerbos/cerbos/blob/ff338f84dad041d0676ed9abaf2b39e45eb54719/internal/storage/index/builder.go#L125) to add the source file information that we later use during compilation)
- After doing the above, the policies returned by the `GetPolicy` Admin API is guaranteed to have the store identifier in the metadata (add tests to make sure of that)
- Now the client side implementation is much simpler because all the required information is contained within the policy object and you don't have to create new structs and copies of the returned list in order to sort/filter by fields.